bigchaindb/tendermint/core.py

```diff
@@ -62,7 +62,7 @@ def check_tx(self, raw_transaction):
             logger.debug('check_tx: INVALID')
             return Result.error()
 
-    def begin_block(self, block_hash, header):
+    def begin_block(self, reqBeginBlock):
```

Comment:
please use `snake_case`

Reply:
Maybe also write something about "reqBeginBlock" in the docstring.